packages/core/src/Utils/Language/language.js

```diff
@@ -25,5 +25,6 @@ export const getURL = lang => urlForLanguage(lang);
 export const changeLanguage = key => {
     SocketCache.clear();
 
+    window.localStorage.setItem('i18n_language', key);
```

Comment:
can we check if its english here, avoiding setting it up twice, and pls check the uppercasing 🙏

Reply:
Thanks. the condition is added.